### DISCUSSION TITLE
New Intents

### DISCUSSION THREAD

**Discussion by synesthesiam** [OHF-Voice/intents#1481] [Announcements]
Hi language leaders! :wave: 

We have recently added support for the `HassShoppingListAddItem` intent in Assist, and are ready to accept translations :slightly_smiling_face:  You can see the [English example here](https://github.com/home-assistant/intents/blob/main/sentences/en/shopping_list_HassShoppingListAddItem.yaml).
This is a pretty simple intent: just add an item to a shopping list and response that it was added. It uses the new "wildcards" feature, which you can see [in the _common.yaml](https://github.com/home-assistant/intents/blob/7ff9882abb827d7e0cafc9e4127ac8e5bffba547/sentences/en/_common.yaml#L289).

As we add more support for intents (temperature is next), I will be announcing them here. Thank you for all of your work :rocket: 

@AalianKhan @Ahmed-farag36 @Natfaji @al-one @alpdmrel @Arkkimaagi @arunshekher @auanasgheps @xraver @bluefoxlee @charlesdagenais @chuk-a @cibernox @cvladan @davefx @dinhchinh82 @dontinelli @dsimop @duhow @easterapps @mib1185 @ErnestStaug @flexy2dd @gabimarchidan @tetele @haim-b @leranp @halecivo @schizza @HepoH3 @hertzg @hristo-atanasov @joaorgoncalves @nikito7 @larsdunemark @LaStrada @limitless00net @LubosKadasi @makstech @mojikosu @MTrab @fadamsen @nagyrobi @skynetua @spuljko @tetele @thdg @TheFes @Uriziel01 @witold-gren @zubir2k

**Comment by tetele** [comment]
I would suggest tagging and publishing the intents repo as late as possible before the HA build, so that as many languages get a chance to add the new intent in 2023.8.

**Reply by frenck** [reply]
We will tag another one if needed/turns out to be helpful.

**Comment by duhow** [comment]
PR example to help: https://github.com/home-assistant/intents/pull/1489

**Comment by synesthesiam** [comment]
@home-assistant/language-leaders I will be tagging a new intents release tomorrow (December 5th) for the December release of Home Assistant.

There are several new intents in this release that need translations:

* `HassNevermind` - cancel a request (no response)
    * "never mind"
* `HassGetWeather` - get the current temperature/condition (first weather entity or by name)
    * "what is the weather"
    * "what is the weather for &lt;name&gt;"
* `HassListAddItem` - add an item to a to-do list by name
    * "add &lt;item&gt; to &lt;name&gt;"
* `HassClimateGetTemperature` - get the current inside temperature (first climate entity or by name/area)
    * "what is the temperature"
    * "what is the temperature for &lt;name&gt;"
    * "what is the temperature in &lt;area&gt;"

No big rush since we can always update in a point release, but it would be great to have as much support as possible :slightly_smiling_face:

**Reply by TheFes** [reply]
Thanks for the heads up, I'll try to add the missing parts for NL tomorrow

**Reply by tetele** [reply]
For those who use a comma as a decimal separator (`123,4`), a heads up: the TTS for your language may not generate numbers correctly if they are formatted using a dot, as they are now (`123.4`). Given the new temperatures for weather and climate that HA can now utter, the number of occurences of this issue may increase.

For Romanian, I've fixed it [like this](https://github.com/home-assistant/intents/pull/1706), but I'm looking for a cleaner solution.

**Reply by skynetua** [reply]
@tetele I believe this should be fixed on core side. Service returning state should format numbers in the way defined in HA settings.

**Reply by TheFes** [reply]
Sorry work was too busy to add it, NL will have to wait for a later release

**Reply by andrejs2** [reply]
Thanks @tetele for info. Slovenian uses comma as well and in Slovenian it's pronouced  like "1.4 grade" - > "ena cela štiri stopinje". I tried [here](https://github.com/home-assistant/intents/blob/main/responses/sl/HassClimateGetTemperature.yaml) and still have to improve the response for negative format.

But should be OK for spoken output of decimals, if it is written output (in Assist dialog box f.i.) it's funny to have answer like "one point four grade".

**Reply by tetele** [reply]
@skynetua i agree that the core should expose a Jinja filter for this, but locale settings are at the frontend level (i.e. you could have a different language on each device, which could be different from the one of the voice pipeline), so the other part can't be implemented.

**Reply by tetele** [reply]
Or maybe i misunderstood and you meant to convert it based on pipeline locale

**Reply by skynetua** [reply]
@tetele you are right, pipeline locale should be used.

**Comment by LubosKadasi** [comment]
THX 4 info & examples,

+ Slovakian also use "," instead of ".". All done 4 now. Will wait 4 update.

LK.
